tweaks.py

```diff
@@ -60,12 +65,110 @@
+    """Try to create hardlink (copy file as fallback)"""
+    try:
+        os.link(src, dst)
+    except OSError:
+        shutil.copy(src, dst)
+
+
+class Tweaks:
     """
     Class grouping tweaks for a specific game
     """
 
     def __init__(self, appid):
         self.env = {}
+        self.needs_fonts = False
+        self.prefix = os.environ['STEAM_COMPAT_DATA_PATH'] + '/pfx/'
         if appid in TWEAKS_DB:
             self.env = TWEAKS_DB[appid].get('env') or {}
+            self.needs_fonts = TWEAKS_DB[appid].get('needs_fonts') or False
+
+
+    def import_fonts(self):
+        """Import missing fonts to game's Wine prefix
+
+        This function will query fontconfig looking for specific fonts,
+        otherwise it will look into ~/.wine/ and finally into /usr/share/fonts/.
+
+        Installation consists only of putting files into default Windows font
+        directory - there's no need to use regedit (unlike in winetricks), Wine
+        will detect and register fonts on next run.
+
+        Fonts will probably not be recognized if installed while application is
+        running in Wine.
+        """
+
+        ms_core_fonts = [
+            ('Arial', '', 'arial.ttf'),
+            ('Arial', 'Bold', 'arialbd.ttf'),
+            ('Arial', 'Italic', 'ariali.ttf'),
+            ('Arial', 'Bold Italic', 'arialbi.ttf'),
+            ('Arial Black', '', 'ariblk.ttf'),
+            ('Comic Sans MS', '', 'comic.ttf'),
+            ('Comic Sans MS', 'Bold', 'comicbd.ttf'),
+            ('Courier New', '', 'cour.ttf'),
+            ('Courier New', 'Bold', 'courbd.ttf'),
+            ('Courier New', 'Italic', 'couri.ttf'),
+            ('Courier New', 'Bold Italic', 'courbi.ttf'),
+            ('Georgia', '', 'georgia.ttf'),
+            ('Georgia', 'Bold', 'georgiab.ttf'),
+            ('Georgia', 'Italic', 'georgiai.ttf'),
+            ('Georgia', 'Bold Italic', 'georgiaz.ttf'),
+            ('Impact', '', 'impact.ttf'),
+            ('Times New Roman', '', 'times.ttf'),
+            ('Times New Roman', 'Bold', 'timesbd.ttf'),
+            ('Times New Roman', 'Italic', 'timesi.ttf'),
+            ('Times New Roman', 'Bold Italic', 'timesbi.ttf'),
+            ('Trebuchet MS', '', 'trebuc.ttf'),
+            ('Trebuchet MS', 'Bold', 'trebucbd.ttf'),
+            ('Trebuchet MS', 'Italic', 'trebucit.ttf'),
+            ('Trebuchet MS', 'Bold Italic', 'trebucbi.ttf'),
+            ('Verdana', '', 'verdana.ttf'),
+            ('Verdana', 'Bold', 'verdanab.ttf'),
+            ('Verdana', 'Italic', 'verdanai.ttf'),
+            ('Verdana', 'Bold Italic', 'verdanaz.ttf'),
+            ('Webdings', '', 'webdings.ttf')
+        ]
+        for font in ms_core_fonts:
+            if not self.is_font_installed(font[0]):
+                self._install_ttf_font(*font)
+
+
+    def is_font_installed(self, file_name):  # pylint: disable=missing-docstring
+        return os.access(self.prefix + '/drive_c/windows/Fonts/' + file_name, os.F_OK)
```

Comment:
os.path.join

Reply:
I know `os.path.join` is an idiomatic way of creating paths in Python, but:

1) Rest of code uses string concatenation for this purpose
2) This code is not cross-platform, it is intended to run on Unix systems only
3) There are no user-supplied strings concatenated here - we can expect prefix and file_name to be properly escaped already (this is a shaky argument)

*I am not ruling out using os.path.join, merely explaining why I didn't use it for this PR.*

Reply:
I'd say the more interesting usage of os.path.join is avoiding double-path-separators. Forward slash works just fine on Windows

Reply:
@plaes @nanonyme - implemented, thank you for review :)